Firestore/Example/Tests/SpecTests/FSTSpecTests.mm

```diff
@@ -743,8 +743,8 @@ - (void)runSpecTestSteps:(NSArray *)steps config:(NSDictionary *)config {
     for (NSDictionary *step in steps) {
       LOG_DEBUG("Doing step %s", step);
       [self doStep:step];
-      [self validateStepExpectations:step[@"expect"]];
-      [self validateStateExpectations:step[@"stateExpect"]];
+      [self validateStepExpectations:step[@"expectedSnapshotEvents"]];
```

Comment:
Can you rename `validateStepExpectations` to `validateExpectedSnapshotEvents` or similar? (and any other similarly-named methods / variables you stumble across along the way)